src/main/java/types/CpuPinningPolicy.java

```diff
@@ -69,6 +69,18 @@
      * @status added
      * @since 4.5
      */
-    RESIZE_AND_PIN_NUMA;
+    RESIZE_AND_PIN_NUMA,
+
+    /**
+     * The CPU pinning will be automatically calculated by the engine when a vm starts and it will be dropped when the vm stops.
+     *
+     * The pinning is exclusive, that means that no other VM can use the pinned pCPU.
```

Comment:
pCPU -> physical CPU

Reply:
Lets resolve the typo, otherwise approved